Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/EditorVertexSelectionBus.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+
+namespace AzToolsFramework
+{
+    //! Bus to request operations to the EditorVertexSelection classes.
+    class EditorVertexSelectionRequests : public AZ::EBusTraits
+    {
+    public:
+        virtual void DuplicateSelectedVertices() {};
+        virtual void DeleteSelectedVertices() {};
+        virtual void ClearVertexSelection() = 0;
+
+    protected:
+        ~EditorVertexSelectionRequests() = default;
+    };
+    using EditorVertexSelectionRequestBus = AZ::EBus<EditorVertexSelectionRequests>;
```

Comment:
update: I think we should be able to use the already existing buses for this - please see `VertexContainerInterface` for the existing buses (`FixedVerticesRequestBus` and `VariableVerticesRequestBus`). This new type should be removed.

Reply:
I have checked those buses and I'm not sure how they can be used without an intermediary. May need some assistance here.

Reply:
Sure thing, give me a ping later and we can take a look 👍